rosdep/python.yaml

```diff
@@ -2493,6 +2493,20 @@ python-levenshtein:
   fedora: [python-Levenshtein]
   gentoo: [dev-python/python-levenshtein]
   ubuntu: [python-levenshtein]
+python-pylibdmtx-pip: &migrate_eol_2025_04_30_python3_pylibdmtx_pip
+  debian:
+    pip:
+      packages: [pylibdmtx]
+  fedora:
+    pip:
+      packages: [pylibdmtx]
+  osx:
+    pip:
+      packages: [pylibdmtx]
+  ubuntu:
```

Comment:
Similarly, this looks like it is available in Ubuntu since groovy: https://packages.ubuntu.com/groovy/python3-pylibdmtx .  Now this is a trickier one; there are no Ubuntu LTS releases that this is currently in.  But even with that, I think we should put a wildcard entry for the system package on Ubuntu, and override it with `pip` for bionic and focal.